ui/js/directives/treeherder/clonejobs.js

```diff
@@ -197,12 +197,20 @@ treeherder.directive('thCloneJobs', [
                     });
                     if (isGroupExpanded(gi.jgObj)) {
                         gi.jgObj.groupState = "collapsed";
-                        addGroupJobsAndCounts(gi.jgObj, gi.platformGroupEl);
+                        // fade out to hidden, then show again
+                        gi.grpJobList.fadeOut("fast", function() {
```

Comment:
the action is a little different than it was before.  It looks like it takes a bit of time to fade-out, then snaps in.  I wonder if we could skip the fade-out part here and just snap away the jobs and fade-in the groups?